test/t8_forest/t8_gtest_forest_new_from_part_cmesh.cxx

```diff
@@ -0,0 +1,125 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element classes in parallel.
+
+  Copyright (C) 2015 the developers
```

Comment:
Maybe change the date to 2024?

Reply:
Yes, I already have the template for VScode that creates the header for new files, but this file was still created via copy paste before, so I will change it

Reply:
How do you create a template for these things in VS Code (I'm curious because I use that editor too)?